### PR TITLE
Updated Swagger for IA-Connect 9.3 release (JML connector)

### PR DESCRIPTION
**Pull request by SimonBondyUltima**
Updated Swagger for IA-Connect 9.3 release: JML connector

NOTE: For all actions that have new inputs, the API acts as per the previous version if you don't provide this inputs. This means that existing flows will not break. This has been extensively tested.
NOTE: For all actions that have new outputs, these are in addition to the existing outputs and simply make new flows simpler (by providing you with more information) but does not impact existing flows. Again this has been extensively tested.

ActiveDirectoryAddADUser has new outputs: CreatedUserDistinguishedName, CreatedUserSAMAccountName, CreatedUserPrincipalName
ActiveDirectoryAddADUserToMultipleADGroupsByName has new input MaxGroupsPerCall
ActiveDirectoryResetADUserPasswordByIdentity has new input ResetPasswordTwice
ActiveDirectoryRemoveADUserFromMultipleADGroupsByNamehas new input MaxGroupsPerCall
OpenExchangePowerShellRunspace has new inputs OnlyConnectIfNotAlreadyConnected, CommandTypesToImportLocally, AdditionalCommandsToImportLocallyCSV
IsExchangePowerShellRunspaceOpen has new input TestCommunications
Added new action ExchangeSetMailboxAutoReplyConfiguration
Added new actions AzureADv2AddAzureADUser, AzureADv2ResetAzureADUserPassword
AzureADv2GetAzureADUserGroupMembership has new input PropertiesToReturn
AzureADv2AddUserToGroup and AzureADv2RemoveUserFromGroup have a default changed for the input CheckUserGroupMembershipsFirst (was false, now true). This is was changed because the Swagger was incorrect: The default for the action was actually true but accidently set to false in the previous Swagger.
Added new actions AzureADv2AddADUserToMultipleADGroups, AzureADv2RemoveADUserFromMultipleADGroups
AzureADv2RemoveUserFromAllGroups has new input MaxAzureADGroupsPerCall
AzureADv2SetAzureADUser has new inputs FaxNumber, AgeGroup, ConsentProvidedForMinor, MailNickName, EmployeeId
Added new actions AzureADv2ResetAzureADUserProperties, AzureADv2NewSecurityGroup, AzureADv2RemoveSecurityGroup, AzureADv2NewMicrosoft365Group, AzureADv2GetGroups, AzureADv2EnableUser, AzureADv2DisableUser
Actions OpenO365PowerShellRunspace and OpenO365PowerShellRunspaceWithCertificate have new inputs OnlyConnectIfNotAlreadyConnected, CommandTypesToImportLocally, AdditionalCommandsToImportLocallyCSV
Action IsO365PowerShellRunspaceOpen has new input TestCommunications
Action O365NewO365DistributionGroup has new outputs CreatedGroupDN, CreatedGroupGUID, CreatedGroupIdentity
Added new actions O365RemoveDistributionGroup, O365SetO365MailboxAutoReplyConfiguration
Added new action JMLConnectToJMLEnvironment

---
### When submitting a connector, please make sure that you follow the requirements below, otherwise your PR might be rejected. We want to make you have a well-built connector, a smooth certification experience, and your users are happy :) 

If this is your first time submitting to GitHub and you need some help, please sign up for this [session](https://forms.office.com/pages/responsepage.aspx?id=KtIy2vgLW0SOgZbwvQuRaXDXyCl9DkBHq4A2OG7uLpdUMTFJWFFGVUxBNUFZQjZWRUdaOE5BMFkwNS4u). 

- [x] I attest that the connector doesn't exist on the Power Platform today. I've verified by checking the pull requests in GitHub and by searching for the connector on the platform or in the documentation. 
- [x] I attest that the connector works and I verified by deploying and testing all the operations. 
- [x] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [x] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [x] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [x] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [ ] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [ ] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
- [ ] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
- [ ] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md.

### CODE REVIEW

**Comment by v-EgorKozhadei** [comment]
Hello @SimonBondyUltima!

Your PR is being reviewed. We will get back with feedback soon.

File: certified-connectors/IA-Connect JML/apiDefinition.swagger.json [hunk 1]
```diff
@@ -6685,167 +6882,1261 @@
...
+            "name": "AzureADv2AddUserToGroup",
+            "description": "Inputs for IA-Connect AzureADv2AddUserToGroup action",
+            "required": true,
+            "schema": {
+              "type": "object",
+              "properties": {
+                "UserObjectId": {
+                  "type": "string",
+                  "title": "User object Id or UPN",
+                  "description": "The Id of an Azure Active Directory user. You can specify a user by UPN (e.g. user@mydomain.onmicrosoft.com) or ObjectId (e.g. 01234567-0123-0123-0123-0123456789ab)."
```
Comment (v-EgorKozhadei) [inline certified-connectors/IA-Connect JML/apiDefinition.swagger.json:7386]: @SimonBondyUltima secre-look-a-like-values are not allowed, there're a few of them in descriptions. Could you please replace them with some string value? E.g. `ObjectId (e.g. UUID/GUID value)`

File: certified-connectors/IA-Connect JML/apiDefinition.swagger.json [hunk 2]
```diff
@@ -7022,49 +8550,52 @@
...
               "properties": {
+                "ObjectId": {
+                  "type": "string",
+                  "title": "User object Id or UPN",
+                  "description": "The Id of an Azure Active Directory user. You can specify a user by UPN (e.g. user@mydomain.onmicrosoft.com) or ObjectId (e.g. 01234567-0123-0123-0123-0123456789ab)."
+                },
+                "Manager": {
+                  "type": "string",
+                  "title": "Manager",
+                  "description": "To add a manager, specify the user's manager Object Id (e.g. a7199172-4c2c-41e5-bed8-bd7a3a1303a5) or UPN (e.g. MyBoss@MyDomain.com). To remove the user's manager, set this field to blank."
```
Comment (v-EgorKozhadei) [inline certified-connectors/IA-Connect JML/apiDefinition.swagger.json:8597]: @SimonBondyUltima this description also has a secret-look-a-like-value, different from the previous comment. Please replace it too.

**Review by v-EgorKozhadei — COMMENTED** [review]
Hello @SimonBondyUltima!

Please check comments after the review. There are multiple occurrences of secret-look-a-like-values. They can be removed or replaced with a string value.

Let me know if you have any questions.

**Comment by SimonBondyUltima** [comment]
Hi,

Thanks for the comments.

These are all made-up values. For example: 01234567-0123-0123-0123-0123456789ab
These are here to give you an example of what the format should look like. I did consider simply saying “GUID format” but I have several customers who don’t know what that means, so I started giving actual examples (obviously made-up).
But are you saying that I cannot even use made-up values such as the one above?

Admittedly a7199172-4c2c-41e5-bed8-bd7a3a1303a5 does look less made-up, but it was created in a GUID generator and doesn’t correspond with something real.

SIMON​​

BOND
 |
Lead Developer
 |
www.ultima.com<http://www.ultima.com/>
 |
+443330158632<tel:+443330158632>

**Comment by v-EgorKozhadei** [comment]
Hello @SimonBondyUltima,

Thank you for your message!

Unfortunately, we can't use secret-look-a-like-values even if they're made up.
There are only two options: either deleting the value or replacing it with a string value.

If you want, you can use an external link in intro.md file for sharing some instructions about possible values and their formats used by the connector.

**Comment by SimonBondyUltima** [comment]
Hi, I've removed all example SID and GUID values from the descriptions.

**Review by v-EgorKozhadei — APPROVED** [review]

**Comment by v-EgorKozhadei** [comment]
Hello @SimonBondyUltima,

Congratulations, your PR is approved and merged.

Please update your submission in ISV Studio by using the latest commit ID.

Let us know if you have any questions.

**Pull request merged**